.asf.yaml

```diff
@@ -24,11 +24,14 @@ github:
     - maven-plugins
     - maven-release-plugin
     - maven
+  protected_branches:
+    master: { }
+  pull_requests:
+    del_branch_on_merge: true
   enabled_merge_buttons:
     squash: true
     merge: false
     rebase: true
-  del_branch_on_merge: true
   features:
     issues: true
 notifications:
```

Comment:
I know, it was not part of the task, but please uncomment the two lines in the notification section.